### ISSUE TITLE
Taxes and Shipping is unaccessible when allowAdminChanges is set to false

### ISSUE THREAD

**Issue by echantigny** [craftcms/commerce#701]
### Description

The taxes and shipping rates are not saved in the project config file, but it is impossible to get to them while the allowAdminChanges is set to false, getting a 403 Forbidden.

I understand that changing the shipping rates is something that shop owners might want to do on their own, but it could also be locked behind the project configs because those rarely change.

Taxes don't change that much, in Canada at least.  So these could also be locked behind the project configs.

Either way, right now, it's impossible to set them in an environment using the config file or the CP.   

### Steps to reproduce

```
'useProjectConfigFile' => true,
'allowAdminChanges' => false,
```

### Additional info

- Craft version: Craft Pro 3.1.8
- PHP version: 7.2.4
- Database driver & version: MySQL 5.7.21
- Plugins & versions: Commerce 2.0.4

**Comment by davist11** [comment]
Wanted to bump this and see what the plan was for addressing this? Otherwise we are going to unfortunately have to disable project config.

**Comment by echantigny** [comment]
Also seems like Gateways are also part of this issue.  They do not propagate on sync, but they are not accessible.

**Comment by andris-sevcenko** [comment]
> Also seems like Gateways are also part of this issue. They do not propagate on sync, but they are not accessible.

Will look into this part when I’m back next week. Gateways should be fully supported for Project Config

**Comment by echantigny** [comment]
@andris-sevenko I think you are right, and they do propagate, but in a weird way.  Might just be an issue of having a site that didn't have the project config to start with.  I ended up with 2 of the same gateway and the dummy that I just disabled on dev was not copied over, which I had completely deleted on the staging site.  I don't think you need to worry much about this one.

**Comment by lukeholder** [comment]
We are needing to shuffle around the setting page in the CP to expose shipping and taxes. Will be fixed soon.

**Comment by echantigny** [comment]
This is now fixed in 2.1
